libcodechecker/analyze/analyzers/result_handler_plist_to_db.py

```diff
@@ -58,48 +58,49 @@ def __store_bugs(self, files, bugs, connection, analisys_id):
                 connection.add_file_content(file_descriptor.fileId,
                                             compressed_file)
 
-        # Skipping bugs in header files handled here.
+        # Skipping reports in header files handled here.
         report_ids = []
-        for bug in bugs:
-            events = bug.events()
+        for report in reports:
+            events = report.events()
 
             # Skip list handler can be None if no config file is set.
             if self.skiplist_handler:
                 if events and self.skiplist_handler.should_skip(
-                        events[-1].start_pos.file_path):
-                    # Issue #20: this bug is in a file which should be skipped
-                    LOG.debug(bug.hash_value + ' is skipped (in ' +
-                              events[-1].start_pos.file_path + ")")
+                        events[-1].location.file_path):
+                    # Issue #20: this report is in a file which
+                    # should be skipped
+                    LOG.debug(report.hash_value + ' is skipped (in ' +
+                              events[-1].location.file_path + ")")
                     continue
 
             # Create remaining data for bugs and send them to the server.
             bug_paths = []
-            for path in bug.paths():
+            for path in report.paths():
```

Comment:
This is kept this way to be backward compatible with the current way of storing path information. It will be changed in the upcoming schema rewriting.